### ISSUE TITLE
Rename "Import as asset" to something more obvious

### ISSUE THREAD

**Issue by krzysiula3000** [Sollumz/Sollumz#967]
eg. "import to asset library" because right now people confuse game assets with blender assets

**Comment by krzysiula3000** [comment]
i couldve opened a pr now that i think about it but already too late lol

**Comment by gg781** [comment]
Choose your personality:

- *\*Add to Blender asset browser\**  
- *\*Only silly enable this\**  
- *\*Are you ass-by-stupid don't enable if you don't know what does import as asset means\**  
- *\*Donate the file to the asset library\**  
- *\*meow meom\**

**Comment by krzysiula3000** [comment]
why are you crying in a bug report over actions i didnt even do what is yall problem bruh

**Comment by krzysiula3000** [comment]
choose your mental disorder:

* attacking random people trying their best to help
* trying to be edgy wayyyy too hard
* hypocrisy
* abusing sarcasm to the absolute limits (sarcasm/cringe border)

**Comment by ook3D** [comment]
> choose your mental disorder:
> 
> * attacking random people trying their best to help
> * trying to be edgy wayyyy too hard
> * hypocrisy
> * abusing sarcasm to the absolute limits (sarcasm/cringe border)

I think you're taking this way too serious, ggs was making a light hearted joke

**Comment by krzysiula3000** [comment]
is that ggs i thought it was one of them

**Comment by gg781** [comment]
> why are you crying in a bug report over actions i didnt even do what is yall problem bruh
> 
> also this salty sarcasm only makes you look goofy sir

Yeah, sorry, was too exciting that breaks the red line at that time. This should be a serious place to discuss about the project.  

And the wrong word chose to use made people think I was insulting, sorry again, I was mean that the list options can be choose to tell people what does the option do.

**Comment by krzysiula3000** [comment]
yeah its ok mb gang i just thought it was one of them trying to make themselves sound intelligent again

**Comment by Hancapo** [comment]
based ggs
